en/docs/setup/running-the-product.md

```diff
@@ -127,158 +116,68 @@ the Command Prompt as described below.
         $args \> /dev/null 2\>&1 &**
         
         !!! tip
-            The only change is replacing `             sh            ` with
-            `             bash            ` . This is required only for Solaris.
-    4.  Start the product by following the above instructions.
-
-### Accessing the management console
+            Replace `sh` with `bash` in above occurences.
+    4.  Start the product.
 
-Once the server has started, you can access the Management Console by
-opening a Web browser and typing in the management console's URL. The
-URL is displayed towards the end of the server start script's console
-and log. For example:
+## Accessing the management console
 
-![](attachments/34611935/44179743.png) 
+Once the server has started, you can access the Management Console by navigating 
+to the URL of the management console . The URL is displayed towards the end of the 
```

Comment:
remove the extra space between "management console" and the fullstop.

Reply:
fixed: ba2206a14a2b5d1c659c68b00e4594b5771d414d